corporate/lib/stripe.py

```diff
@@ -1276,49 +1278,65 @@ def create_card_update_session(self) -> Dict[str, Any]:
     def apply_discount_to_plan(
         self,
         plan: CustomerPlan,
-        discount: Decimal,
+        customer: Customer,
```

Comment:
Isn't this just `self.get_customer()`? I think we generally try to avoid passing that as a parameter.

Reply:
Yeah, I can just modify it to use `plan.customer`.